src/block/tile/BrewingStand.php

```diff
@@ -108,4 +126,125 @@ public function getInventory(){
 	public function getRealInventory(){
 		return $this->inventory;
 	}
+
+	private function checkFuel(Item $item) : void{
+		$ev = new BrewingFuelUseEvent($this);
+		$ev->call();
+
+		if($ev->isCancelled()) {
+			return;
+		}
+
+		$item->pop();
+		$this->inventory->setFuel($item);
+
+		$this->maxFuelTime = $this->remainingFuelTime = $ev->getFuelTime();
+	}
+
+	private function canBrew() : bool{
+		if($this->inventory->getIngredient()->isNull()){
+			return false;
+		}
+
+		for($i = 1; $i <= 3; ++$i){
+			$input = $this->inventory->getItem($i);
+			if($input->isNull()) {
+				continue;
+			}
+
+			if($this->position->getWorld()->getServer()->getCraftingManager()->matchBrewingRecipe($input, $this->inventory->getIngredient()) !== null){
+				return true;
+			}
+		}
+
+		return false;
+	}
```

Comment:
I think this should return an array of the brewable slots. When I first saw this function, I mistook it for a general "can anything be brewed" method, and that could easily be accidentally used incorrectly in future refactors.